tests/cases/unittests/jsDocParsing.ts

```diff
@@ -998,7 +1001,10 @@ module ts {
                         ? JSON.parse(Utils.sourceFileToJSON(v))
                         : v;
                 }, 4);
-                
+
+                // Remove the parserContextFlags from the comparison
+                result = result.replace(/\,\n\s+\"parserContextFlags\": \"JavaScriptFile\"/g, "");

```

Comment:
It's not that branch, it's line 287, and the comment explicitly states *"We only care what the parser set directly on the ast"*, which these are, so it seems even more hacky (if that's possible) to explicitly drop this bit there.